CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+## 5.8.0
```

Comment:
We'll update versions and changelog before releases, so you don't have to update them.

Reply:
got it. I will remove this commit! :)